x-pack/plugins/canvas/canvas_plugin_src/functions/common/csv.test.ts

```diff
@@ -37,6 +38,8 @@ describe('csv', () => {
 one,1
 two,2
 fourty two,42`,
+          delimiter: '',
```

Comment:
What happened to cause all of these changes? Doesn't seem like this is involved in any of the image changes?

Reply:
Before it was not passing the `CheckTypes` CI step. Right now, everything is ok, I've reverted changes.

Reply:
Done.